src/evict/evict_lru.c

```diff
@@ -616,23 +625,11 @@ __evict_update_work(WT_SESSION_IMPL *session)
      * trigger.
      */
     if (bytes_inuse < (uint64_t)((target + trigger) * bytes_max) / 200) {
-        if (dirty_inuse < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200)
+        if (bytes_dirty < (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200)
             LF_SET(WT_CACHE_EVICT_SCRUB);
     } else
         LF_SET(WT_CACHE_EVICT_NOKEEP);
 
-    /*
-     * Try history store evict when:
-     * (1) the cache is stuck; OR
-     * (2) the history store score goes over 80; and
-     * (3) the cache is more than half way from the dirty target to the
-     *     dirty trigger.
-     */
-    if (__wt_cache_stuck(session) ||
-      (__wt_cache_hs_score(cache) > 80 &&
-          dirty_inuse > (uint64_t)((dirty_target + dirty_trigger) * bytes_max) / 200))
-        LF_SET(WT_CACHE_EVICT_HS);
-
```

Comment:
This is unrelated, but I was adding `WT_CACHE_EVICT*` flags, and as far as I can see this one is no longer needed.